specification/sse/src/main/scripts/org/kaazing/specification/sse/unnamed.event/multiple.events/response.rpt

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property date "Mon, 1 Jan 2015 01:23:45 GMT"
+
+accept tcp://localhost:8000
+accepted
+connected
+
+read "GET /sse HTTP/1.1\r\n"
+read "Host: localhost:8000\r\n"
+read "Accept: text/event-stream\r\n"
+read "\r\n"
+
+write "HTTP/1.1 200 OK\r\n"
+write "Content-Type: text/event-stream\r\n"
+write "Cache-Control: no-cache\r\n"
+write "Connection: keep-alive\r\n"
+write "Date: " ${date} "\r\n"
+write "\r\n"
+
+write "data:Message for unnamed server-sent event!\n"
```

Comment:
Consider distinguishing the first and last message here.  Currently they both are "Message for unnamed server-sent event!"